parameter_files/archive/new_shrubPFT_patch_params.xml

```diff
@@ -0,0 +1,161 @@
+      <fates_recruit_init_density> 0.16 </fates_recruit_init_density>
+    </pft_group>
+
+
+    <!-- Override values for the new colddecid arctic PFT, this is now PFT 11 -->
+    <!-- which duplicated the orginial 9th PFT -->
+    <pft_group ids="11">
+      <fates_allom_d2bl1> 0.0481934 </fates_allom_d2bl1>
+      <fates_allom_d2bl2> 1.7176758 </fates_allom_d2bl2>
+      <fates_allom_agb2> 0.6853945 </fates_allom_agb2>
+      <fates_allom_agb3> 1.7628613 </fates_allom_agb3>
+      <fates_allom_d2h1> 0.7781250 </fates_allom_d2h1>
+      <fates_allom_d2h2> 0.4027002 </fates_allom_d2h2>
+      <fates_recruit_height_min> 0.8 </fates_recruit_height_min>
+      <fates_allom_dbh_maxheight> 1.9 </fates_allom_dbh_maxheight>
+      <fates_recruit_seed_dbh_repro_threshold> 1.9 </fates_recruit_seed_dbh_repro_threshold>
+      <fates_allom_d2ca_coefficient_min> 0.3166497 </fates_allom_d2ca_coefficient_min>
+      <fates_allom_d2ca_coefficient_max> 0.3166497 </fates_allom_d2ca_coefficient_max>
+      <fates_alloc_store_priority_frac> 0.6 </fates_alloc_store_priority_frac>
+      <fates_alloc_storage_cushion> 1.4 </fates_alloc_storage_cushion>
+      <fates_leaf_slatop> 0.032 </fates_leaf_slatop>
+      <fates_leaf_slamax> 0.032 </fates_leaf_slamax>
+      <fates_leaf_vcmax25top> 54 </fates_leaf_vcmax25top>
+      <fates_mort_scalar_hydrfailure> 0.8 </fates_mort_scalar_hydrfailure>
+      <fates_mort_scalar_cstarvation> 0.57 </fates_mort_scalar_cstarvation>
+      <fates_mort_scalar_coldstress> 3.5 </fates_mort_scalar_coldstress>
+      <fates_mort_bmort> 0.010 </fates_mort_bmort>
+      <fates_mort_freezetol> -95 </fates_mort_freezetol>
+      <fates_frag_seed_decay_rate> 0.46 </fates_frag_seed_decay_rate>
+      <fates_recruit_seed_germination_rate> 0.49 </fates_recruit_seed_germination_rate>
+    </pft_group>
+
+    <!-- Updating parameters for c3_arctic grass based on Yanlan's work. Now PFT 12 -->
+    <pft_group ids="12">
+      <fates_recruit_height_min> 0.11 </fates_recruit_height_min>
+      <fates_alloc_store_priority_frac> 0.6 </fates_alloc_store_priority_frac>
+      <fates_leaf_slatop> 0.027 </fates_leaf_slatop>
+      <fates_leaf_vcmax25top> 86 </fates_leaf_vcmax25top>
+      <fates_grperc> 0.16 </fates_grperc>
+      <fates_mort_scalar_coldstress> 2.3 </fates_mort_scalar_coldstress>
+      <fates_mort_freezetol> -89 </fates_mort_freezetol>
+      <fates_frag_seed_decay_rate> 0.35 </fates_frag_seed_decay_rate>
+      <fates_recruit_seed_germination_rate> 0.29 </fates_recruit_seed_germination_rate>
+    </pft_group>
+
+    <!-- Update the HLM-PFT map since PFTs 9, 10, 11 now share area -->
+    <!-- Splits the HLM deciduous boreal shrub (which must sum to 1.0)  -->
+    <!-- into 0.1 for the evergreen arctic, and 0.8 for cold-decid arctic -->
+    <!-- and leaves 0.1 in the orginial non-arctic extratrop cold-decid shrub -->
+    <pft_group ids="9" >
+      <fates_hlm_pft_map> 0, 0, 0, 0, 0, 0, 0, 0, 0, 0.0, 0.1, 0, 0, 0 </fates_hlm_pft_map>
```

Comment:
here's the problem, we zerod out the 10'th index in this vector. It should have stayed as 1.0

Reply:
@rgknox - Yep, you're exactly right. The 10th index in this vector should have remained as 1.0 (as it always has been in previous versions).  I was only thinking about ELM index 11 changes, and forgot to keep this as is.